src/components/map/Popups/StudyPopup.tsx

```diff
@@ -20,6 +33,12 @@ export default function StudyPopup(record: AirtableRecord) {
         <div className="popup-text">
             {getGeography(record.city, record.state, record.country)}
         </div>
+        <div className="popup-heading">
+            {Translate("PopulationGroup")}
+        </div>
+        <div className="popup-text">
+            {record.population_group ? `${record.population_group}` : Translate("NotReported")}
```

Comment:
Think this should work since `record.population_group` is a string already! If it's null, it'll return the second option - check [this](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator) out for more info
```suggestion
            {record.population_group ?? Translate("NotReported")}
```

Reply:
this makes sense to me as long as we are sure population group will be a string @atatmaja thoughts?

Reply:
`population_group` is typed as a string so it will fail at the ts compiler otherwise anyways, so I am fine with this approach